tests/models/test_models.py

```diff
@@ -166,7 +172,7 @@ def test_scvi(save_path):
     batch = np.zeros(a.n_obs)
     batch[:64] += 1
     a.obs["batch"] = batch
-    setup_anndata(a, batch_key="batch")
+    _setup_anndata(a, batch_key="batch")
```

Comment:
should this be SCVI.setup_anndata?, same with the ones below

Reply:
I didnt want to descrease coverage with the _ setup_anndata since we still have it around. So I oped to use the model-specific setup_anndata call in some spots, and the _ setup_anndata call in other spots

Reply:
would it decrease coverage if it's just calling it behind the scenes? (sorry it's late during this review :) )

Reply:
it shouldn't decrease coverage I believe, code_cov steps through the entire execution

Reply:
This should still be SCVI.setup_anndata right?